kytos/core/interface.py

```diff
@@ -505,6 +768,18 @@ def as_json(self):
         """
         return json.dumps(self.as_dict())
 
+    def notify_link_available_tags(self, controller, src_func=None):
+        """Notify link available tags"""
+        name = "kytos/core.link_available_tags"
+        content = {
+            "interface_id": self.id,
+            "available_tags": self.available_tags,
+            "tag_ranges": self.tag_ranges,
```

Comment:
Do we need really to send tag_ranges here too?

It'd be easier for the topology to only receive `{"interface": self}`

Reply:
Sending only `{"interface": self}` now.